scopesim/optics/optical_train.py

```diff
@@ -222,6 +223,18 @@ def prepare_source(self, source):
         # Convert to PHOTLAM per arcsec2
         # ..todo: this is not sufficiently general
 
+        for ispec, spec in enumerate(source.spectra):
+            # Put on fov wavegrid
+            wave_min = min([fov.meta["wave_min"] for fov in self.fov_manager.fovs])
+            wave_max = max([fov.meta["wave_max"] for fov in self.fov_manager.fovs])
```

Comment:
```suggestion
            wave_min = min(fov.meta["wave_min"] for fov in self.fov_manager.fovs)
            wave_max = max(fov.meta["wave_max"] for fov in self.fov_manager.fovs)
```
Don't need temporary list, just apply `min`/`max` directly to generator expressions.